### DISCUSSION TITLE
FHM model in PyBaMM?

### DISCUSSION THREAD

**Discussion by fred4ets** [pybamm-team/PyBaMM#3040] [Ideas]
Hi there,

Has anyone here heard of the FHM model?

"What if the Doyle-Fuller-Newman model fails? A new macroscale modeling framework"
Harikesh Arunachalam, Member IEEE and Simona Onori, Senior Member, IEEE.

**Comment by valentinsulzer** [comment]
I haven't but it should be easy to test implementing this model in PyBaMM syntax. Just copy this script and edit accordingly https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/full_battery_models/lithium_ion/basic_dfn.py

**Comment by fred4ets** [comment]
Ok, thanks for the prompt answer.

I'll give it a try asap.

**Reply by salman-qadir** [reply]
Hi  @fred4ets fred4ets have you done this  or doing this ?

**Comment by Michael-Pegis** [comment]
I thought I would chime in.....

Its unclear to me what exactly the benefits/assumptions that are fundamentally distinct between FHM and a DFN model where ELY transport properties (thermodynamic factor, [Li]-dependent diffusivity/conductivity).

The author states that 

> ". It must be noted in the FHM model that both the
> electrolyte mass and charge conservation equations consist
> of the electrolyte concentration and potential variables"

But aside from the main table highlighting additional terms in the mass conservation equation that relate to the electrolyte potential, I can't really see any obvious differences. 

![image](https://github.com/pybamm-team/PyBaMM/assets/137376987/88e36178-76f2-488b-b670-d60cbe004b11)

So its unclear whether this paper and a 1D FHM is a dig on "DFN" or a dig on the Brueggemann relationship for spherical particles. As far as I can tell, there isn't anyone stopping you to define effective transport parameters from measured tortuosity/porosity.

But Valentin probably knows better than me here

**Reply by brosaplanella** [reply]
They also seem to define the diffusion in the electrodes alng `x` rather than along `r` (i.e. on the macroscale rather than on the microscale).
